gp-templates/profile-public.php

```diff
@@ -24,7 +24,7 @@
 				}
 			?></dd>
 			<dt><?php _e( 'Member Since', 'glotpress' ); ?></dt>
-			<dd><?php echo date( 'M j, Y', strtotime( $user->user_registered ) ); ?></dd>
+			<dd><?php /* translators: public profile date format */ echo date_i18n( __( 'M j, Y', 'glotpress' ), strtotime( $user->user_registered ) ); ?></dd>
```

Comment:
> A comment makes more sense than `_x()`

Agreed. Context would be redundant, unless the same string is used elsewhere in another context.